app/src/main/java/com/realityexpander/tasky/auth_feature/presentation/splash_screen/MainActivityViewModel.kt

```diff
@@ -15,11 +18,28 @@ import javax.inject.Inject
 @HiltViewModel
 class MainActivityViewModel @Inject constructor(
     private val authRepository: IAuthRepository,
+    private val remindAtNotificationManager : IRemindAtNotificationManager
 ) : ViewModel() {
 
     private val _splashState = MutableStateFlow(SplashState())
     val splashState = _splashState.asStateFlow()
 
+    init {
+        remindAtNotificationManager.createNotificationChannel()
```

Comment:
I wouldnt have the createChannel on the viewmodel. Its hard to find + doesnt make much sense

Imo, the best way is to have the remindAtNotificationManager call it, before it shows the notification


Same with your workerNotification

Reply:
ok, refactor to put this in the class `init` function